src/lib/components/ResultsLoader/ResultsLoader.js

```diff
@@ -25,10 +25,13 @@ ResultsLoader.defaultProps = {
   overridableId: '',
 };
 
-const Element = ({ overridableId }) => (
+const Element = ({ overridableId }) => {
+  const {buildUID} = useContext(AppContext);
```

Comment:
sorry if it was already discussed, but how safe is it to use `buidUID` ? can it be overwritten by mistake in the context?

Reply:
It is pretty safe as the context gets imported from a specific file, so it should be contained to that specific instance, furthermore i think it can not be accessed from outside the RSK component.